src/content/docs/dev/send-tokens/sui/intro.mdx

```diff
@@ -12,26 +12,64 @@ The official Sui integration codebase can be found [here](https://github.com/axe
 
+### Register Coin From Info
+
+If you do not have any metadata available for your coin then you will need to register your token via the `register_coin_from_info()` function. The function returns a new [TokenId](#token-id) and takes five parameters:
+
+1. `its`: The ITS object that will be updated once the new coin is registered.
+1. `name`: The name of the coin being registered.
+1. `symbol`: The symbol of the coin being registered.
+1. `decimals`: The amount of decimals the coin being registered has.
+1. `coin_management`: The type of [management](#coin-management) the coin is registered with.
+
+The function is defined as follows:
+
+```rust
+public(package) fun register_coin_from_info<T>(
+    self: &mut InterchainTokenService,
+    name: std::string::String,
+    symbol: ascii::String,
+    decimals: u8,
+    coin_management: CoinManagement<T>,
+): TokenId {}
+```
+
+This will be a commonly used registration flow as Sui coins often times [freeze](https://docs.sui.io/concepts/object-ownership/immutable#create-immutable-object) its metadata so that no one can alter critical configuration about the coin such as its name, symbol, and decimals.
+
+### Register Coin from Metadata
+
+If you do have the metadata available then you can register the coin via `register_coin_from_metadata()` function. The function returns a new [TokenId](#token-id) and takes three parameters:
+
+1. `its`: The ITS object that will be updated once the new coin is registered.
+1. `metadata` A generic reference to the coin's metadata that contains the coins name, symbol, and decimals.
+1. `coin_management`: The type of [management](#coin-management) the coin is registered with.
+
+```rust
+    public(package) fun register_coin_from_metadata<T>(
+        self: &mut InterchainTokenService,
+        metadata: &CoinMetadata<T>,
+        coin_management: CoinManagement<T>,
+    ): TokenId {}
+```
+
+The coin management must be constructed beforehand, ITS will not construct this automatically for you.
+
+### Register Custom Coin
 To integrate a coin into ITS, the `register_coin()` function must be called. This function takes in the following parameters:
 
-1. `its`: The ITS module that will be updated once the new coin is registered
-1. `coin_info`: The [coin info](#coininfo) represents the of the coin
-1. `coin_management`: The type of [management](#coinmanagement) the coin will have with ITS
+1. `its`: The ITS object that will be updated once the new coin is registered
+1. `coin_info`: The [coin info](#coin-info) represents the of the coin
```

Comment:
> ~~1. `coin_info`: The [coin info](#coin-info) represents the of the coin~~

1. `coin_info`: The [coin info](#coin-info) represents the name, symbol and decimals of the coin